compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -3256,7 +3256,11 @@ declare_lint! {
     /// See the [Checking Conditional Configurations][check-cfg] section for more
     /// details.
     ///
+    /// See the [Cargo Specifics][unexpected_cfgs_lint_config] for configuring this lint in
+    /// `Cargo.toml`.
```

Comment:
What's the reason for specifically calling out `Cargo.toml` here ? and not simply using "Cargo"?

Reply:
If this were about check-cfg, there'd be less of a reason for this to belong here.  I'm specifically wanting to highlight lint configuration which also aligns with my motivation for why this belongs in this section as we generally should call out lint configuration here.